ci/check-crates.sh

```diff
@@ -50,9 +50,11 @@ for file in "${files[@]}"; do
     fi
   done
 
-  result="$(cargo owner --list -q "$crate_name" --token "$CRATE_TOKEN" 2>&1)"
-  if [[ $result =~ ^error ]]; then
-    if [[ $result == *"Not Found"* ]]; then
+  response=$(curl -s https://crates.io/api/v1/crates/"$crate_name"/owners)
```

Comment:
i think we'll be subject to lower api rate-limits without the token, so we need to keep an eye on that